x/ethbridge/querier/querier.go

```diff
@@ -0,0 +1,77 @@
+package querier
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/x/ethbridge/types"
+	keep "github.com/swishlabsco/cosmos-ethereum-bridge/x/oracle/keeper"
+	oracletypes "github.com/swishlabsco/cosmos-ethereum-bridge/x/oracle/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+//query endpoints supported by the oracle Querier
+const (
+	QueryEthProphecy = "prophecies"
+)
+
+// NewQuerier is the module level router for state queries
+func NewQuerier(keeper keep.Keeper, cdc *codec.Codec, codespace sdk.CodespaceType) sdk.Querier {
+	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err sdk.Error) {
+		switch path[0] {
+		case QueryEthProphecy:
+			return queryEthProphecy(ctx, cdc, req, keeper, codespace)
+		default:
+			return nil, sdk.ErrUnknownRequest("unknown ethbridge query endpoint")
+		}
+	}
+}
+
+func queryEthProphecy(ctx sdk.Context, cdc *codec.Codec, req abci.RequestQuery, keeper keep.Keeper, codespace sdk.CodespaceType) (res []byte, err sdk.Error) {
+	var params types.QueryEthProphecyParams
+
+	errRes := cdc.UnmarshalJSON(req.Data, &params)
```

Comment:
ditto with `errRes` and others, should be `err`

Reply:
fixed  - https://github.com/musnit/peggy/commit/68f3f95b4d87e6eb17127bbfc0c90b1988a96ee4 (note, there are still 2 different variables as there are both normal error variables and sdk.Error)